pkg/controller/vsphere/reconciler.go

```diff
@@ -456,15 +458,15 @@ func clone(s *machineScope) (string, error) {
 
 	folder, err := s.GetSession().Finder.FolderOrDefault(s, folderPath)
 	if err != nil {
-		multipleFoundMsg := "multiple folders found, specify one in config"
+		const multipleFoundMsg = "multiple folders found, specify one in config"
 		notFoundMsg := "folder not found, specify valid value"
 		defaultError := fmt.Errorf("unable to get folder for %q: %w", folderPath, err)
 		return "", handleVSphereError(multipleFoundMsg, notFoundMsg, defaultError, err)
 	}
 
 	datastore, err := s.GetSession().Finder.DatastoreOrDefault(s, datastorePath)
 	if err != nil {
-		multipleFoundMsg := "multiple datastores found, specify one in config"
+		const multipleFoundMsg = "multiple datastores found, specify one in config"
```

Comment:
why is this using const and := unconsistently??
https://github.com/openshift/machine-api-operator/pull/605/files#diff-11b6610f82e5d47fdfb8dda44b90dd1eR469
https://github.com/openshift/machine-api-operator/pull/605/files#diff-11b6610f82e5d47fdfb8dda44b90dd1eR470
https://github.com/openshift/machine-api-operator/pull/605/files#diff-11b6610f82e5d47fdfb8dda44b90dd1eR594
https://github.com/openshift/machine-api-operator/pull/605/files#diff-11b6610f82e5d47fdfb8dda44b90dd1eR595